### PR TITLE
fix missing rsod info

### PR DESCRIPTION
**Pull request by TychoVrahe**
adds default title, footer or message if some of these are not set

### CODE REVIEW

File: core/embed/util/rsod/rsod.c [hunk 1]
```diff
@@ -57,9 +58,16 @@ void rsod_terminal(const systask_postmortem_t* pminfo) {
       message = message_buf;
       break;
     case TASK_TERM_REASON_ERROR:
-      title = pminfo->error.title;
-      message = pminfo->error.message;
-      footer = pminfo->error.footer;
+      if (strnlen(pminfo->error.title, sizeof(pminfo->error.title)) != 0) {
```
Comment (cepetr) [inline core/embed/util/rsod/rsod.c:61]: `pminfo->error->xxx` strings are always null-terminated, so we can simplify the condition to `if (pminfo->error.title[0] != '\0') {`.

**Review by cepetr — CHANGES_REQUESTED** [review]

File: core/embed/util/rsod/rsod.c (hunk 1 above)
Comment (TychoVrahe) [reply core/embed/util/rsod/rsod.c:61]: fixed https://github.com/trezor/trezor-firmware/commit/cdb0427a58d6413d64b94adb54bc2107f91cb249

Comment (TychoVrahe) [reply core/embed/util/rsod/rsod.c]: changed the other half here: https://github.com/trezor/trezor-firmware/commit/53d6f1f22bb9a8549c6f84ec6e6817054cf406b7

**Comment by bosomt** [comment]
**QA OK**

```
INTERNAL ERROR

no image buff

PLEASE VISIT TREZOR.IO/RSOD
```

 - Device: Trezor T3T1 2.8.7 regular (revision e196413bb745443610211c325623ba7cee7309e9)

**Pull request merged**